master: Update pkg/testutils/release/cockroach_releases.yaml

cockroach-teamcity opened the pull request:
Update pkg/testutils/release/cockroach_releases.yaml with recent values.

Epic: None
Release note: None
Release justification: test-only updates

rail commented:
bors r-

celiala commented:
> we need the new fixtures to merge this

draft PR for fixtures is here: https://github.com/cockroachdb/cockroach/pull/156535

hopefully this lands today or Monday (wrangling the last set of tests, hopefully 😅 )

cockroach-teamcity committed:
master: Update pkg/testutils/release/cockroach_releases.yaml

Update pkg/testutils/release/cockroach_releases.yaml with recent values.

Epic: None
Release note: None
Release justification: test-only updates

rail commented:
bors r+
